tests/test_timing.py

```diff
@@ -110,16 +110,44 @@ def _default_and_no_exception(log_context):
     (
         timing.logged_duration.default_message,
         (
-            AnyStringMatching(r"Block (executed|raised \w+) in \{duration_real\}s of real-time"),
-            AnyStringMatching(r"Block (executed|raised \w+) in [0-9eE.-]+s of real-time"),
+            {
+                'success': AnyStringMatching(r"Block executed in \{duration_real\}s of real-time"),
+                'error': AnyStringMatching(r"Block raised \w+ in \{duration_real\}s of real-time"),
+            },
+            {
+                'success': AnyStringMatching(r"Block executed in [0-9eE.-]+s of real-time"),
+                'error': AnyStringMatching(r"Block raised \w+ in [0-9eE.-]+s of real-time"),
+            },
+        ),
+    ),
+    (
+        timing.different_message_for_success_or_error(
+            success_message='Block succeeded in {duration_real}s',
+            error_message='Block raised {exc_info[0]} in {duration_real}s',
+        ),
+        (
+            {
+                'success': AnyStringMatching(r"Block succeeded in \{duration_real\}s"),
```

Comment:
you can just use a literal string here, rite? 'snot actually using any re features. Same on line 114.

Reply:
Urrr yep.